client/src/components/AnchorSelectorField/AnchorSelectorField.js

```diff
@@ -16,140 +15,131 @@ import PropTypes from 'prop-types';
 
 const noop = () => null;
 
-class AnchorSelectorField extends SilverStripeComponent {
-  constructor(props) {
-    super(props);
-
-    this.handleChange = this.handleChange.bind(this);
-    this.handleLoadingError = this.handleLoadingError.bind(this);
-  }
-
-  componentDidMount() {
```

Comment:
This was removed as it's unecessary given the componentDidUpdate() useEffect() replacement will also run on mount